specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2022-02-10-preview/desktopvirtualization.json

```
@@ -3772,26 +3772,26 @@
             "values": [
               {
                 "value": "Default",
-                "description": "Agent and other agent side components are delivery schedule is controlled by WVD Infra."
+                "description": "The agent components are updated according to the AVD infrastructure flighting. For more information, see https://aka.ms/avdagent."
               },
               {
                 "value": "Scheduled",
-                "description": "TenantAdmin have opted in for Scheduled Component Update feature."
+                "description": "This enables the Scheduled Agent Updates feature which allows the agent components to get updated according to the schedule that the admin define For more information, see https://aka.ms/avdsau."
```

I forgot to add a period here, my bad. Can you please add
```suggestion
                "description": "This enables the Scheduled Agent Updates feature which allows the agent components to get updated according to the schedule that the admin define. For more information, see https://aka.ms/avdsau."
```